docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
       - 5001:5001
   
   minio:
-    image: minio/minio
+    image: minio/minio:9423-d242680
```

Comment:
do we need to lock this?

Reply:
yeah, didn't work with latest, used from recoracle

Reply:
Ahh, is there a better pinned version that we can use? Like v0.3 or something?  Looking through their I just see stuff like `minio/minio:RELEASE.2019-02-06T21-16-36Z` which is basically the same as what you're doing there (build specific vs minor version specific)

Reply:
I just took this version from recoracle, I guess we could use something else

Reply:
Now it's pinned to the latest release of minio that works.

Reply:
Cool yeah no worries, probably about the same as what you had, they generally have better floating version tags (not in minio's case it looks like)